### ISSUE TITLE
Unable to debug 'SPFX with Angular Elements' using Visual Code

### ISSUE THREAD

**Issue by sagar4u** [pnp/generator-spfx#278]
#### General Information
* [ ] Usage
* [-] Development
* [-] Documentation
* [ ] Feature Request

#### Specific generator
* [ ] Addons
* [ ] HandlebarsJS

#### What Version you are running?
* NodeJS: v10.13
* NPM: v6.4.1
* @pnp/spfx: 2.0.10.
* @pnp/generator-spfx@1.15.0

#### Describe your problem
I am working on SPFX with Angular Elements generated using 'pnp/generator-spfx'.
My question is how to debug this solution using Visual Code and even Chrome Dev tool?

- The debug way mentioned to debug normal SPFX solution using Visual Code doesn't seem to work as it doesn't load source maps.

Any suggestions how to debug Angular Elements?

**Comment by chandaniprajapati** [comment]
Hi @sagar4u ,
If you want to debug only the independent angular element app then you can do it Chrome Dev Tool by searching the file in sources. or you can use VSCode Debugger for Chrome Extension.

If you want to test it with SPFx web part then we can generate the source map on the build of the angular app and then try to use it. though I haven't tried it. I will check this approach and will let you know soon.

**Comment by sagar4u** [comment]
@chandaniprajapati I've tried using both Chrome extension as well as VSCode. Both won't work.
Running angular app alone won't work as we don't get SP context and can't test with SP data.
The reason is I think, we have minified version of angular code without source map in spfx as we have to run 'npm run bundle' which trigger ng prod command.
For debug purpose, we need another command which will give debug mode of angular version in Spfx.
